validator/validator.proto

```diff
@@ -238,6 +238,7 @@ message AmpLayout {
     CONTAINER = 5;
     FILL = 6;
     FLEX_ITEM = 7;
+    INTRINSIC = 8;
```

Comment:
Also there is a validator rule that enforces `width` `height` for `responsive`. Not sure if that is being applied to `intrinsic`, the test to be added above can verify that.